lib/scrabble.rb

```diff
@@ -1,7 +1,20 @@
 class Scrabble
+  attr_reader :score, :word
+
+  def initialize(word)
+    @word = word
+    # @score = score
+  end
 
   def score(word)
-    1
+    letters = word.chars
+    total = 0
+    # point_values.each { |key, value| }
+    letters.each do |letter|
+      if letter == point_values[0]
```

Comment:
`point_values` is a hash, so we should be accessing it here by its keys, not by an index.

`point_values[letter]` will give you the score for any uppercase letter.